install/setup-js-sdk-assets.sh

```diff
@@ -36,5 +36,10 @@ if [[ "${SETUP_JS_SDK_ASSETS:-}" == "1" ]]; then
   # Download those versions & variants using curl
   $dcr --no-deps --rm -v "sentry-nginx-www:/var/www" nginx curl -w '%{response_code} %{url}\n' --no-progress-meter --compressed --retry 3 --create-dirs -fLo "/var/www/js-sdk/#1/#2.min.js" "https://browser.sentry-cdn.com/${versions}/${variants}.min.js" || true
```

Comment:
This will do the trick as said by @BYK in https://github.com/getsentry/self-hosted/pull/3616#pullrequestreview-2684920970.
```suggestion
  $dcr --no-deps --rm -v "sentry-nginx-www:/var/www" --user=nginx nginx curl -w '%{response_code} %{url}\n' --no-progress-meter --compressed --retry 3 --create-dirs -fLo "/var/www/js-sdk/#1/#2.min.js" "https://browser.sentry-cdn.com/${versions}/${variants}.min.js" || true
```

Reply:
I haven't got time to be on my computer again (I'm on mobile now), but IIRC the user should be "www-data" instead of "nginx". But I'll recheck this later tomorrow.

Reply:
I checked and it's nginx.

Reply:
BTW we should check this doesn't get on an old nginx container which already has the files owned by root, as it will then if it wants to get overwritten by nginx user.

I think it's a temporary volume which does not persist during install.sh, but we should make sure.

Reply:
> BTW we should check this doesn't get on an old nginx container which already has the files owned by root, as it will then if it wants to get overwritten by nginx user.
> 
> I think it's a temporary volume which does not persist during install.sh, but we should make sure.

I'm a bit unsure now. Let's try this one first.